packages/admin/admin/src/form/FinalFormFileUpload.tsx

```diff
@@ -82,11 +80,9 @@ const FinalFormFileUploadComponent: React.FunctionComponent<WithStyles<typeof st
     classes,
     disabled,
     dropzoneVariant,
-    caption,
-    multipleFiles = true,
     accept,
     maxSize = 50,
```

Comment:
This is in MB, right?

That may not be flexible enough. It's unlikely but there might be a case where you just want to accept 100kb. So, I'd prefer to do this in bytes